.github/ISSUE_TEMPLATE/new-ontology.yml

```diff
@@ -0,0 +1,249 @@
+  - type: input
+    id: tracker
+    attributes:
+      label: Issue Tracker
+      description: What is the URL of the public issue tracker? Typically, this will be the link to the GitHub repository with a trailing `/issues` URL part.
+      placeholder: ex. https://github.com/obophenotype/uberon/issues
+    validations:
+      required: true
+  - type: input
+    id: download
+    attributes:
+      label: Ontology Download Link
+      description: What URL can be used to download your Ontology?
+      placeholder: ex. https://github.com/obophenotype/uberon/raw/master/uberon.owl
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        # Contact
+
+        This section of the ontology request form contains information about the contact person for the ontology. See also OBO Principle [FP-011 "Locus of Authority"](https://obofoundry.org/principles/fp-011-locus-of-authority.html)
+  - type: input
+    id: contact-name
+    attributes:
+      label: Contact Name
+      description: What is the full name of the contact for the ontology?
+      placeholder: ex. Charles Tapley Hoyt
+    validations:
+      required: true
+  - type: input
+    id: contact-email
+    attributes:
+      label: Contact Email
+      description: What is the email address of the contact for the ontology?
+      placeholder: ex. cthoyt@gmail.com
+    validations:
+      required: true
+  - type: input
+    id: contact-github
+    attributes:
+      label: Contact GitHub Username
+      description: What is the GitHub username of the contact for the ontology? Do not include the @ symbol.
+      placeholder: ex. cthoyt
+    validations:
+      required: true
+  - type: input
+    id: contact-orcid
+    attributes:
+      label: Contact ORCID Identifier
+      description: What is the ORCID identifier of the contact for the ontology? ORCID identifiers can be made at https://orcid.org.
+      placeholder: ex. 0000-0003-4423-4370
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        # Artifacts
+  - type: checkboxes
+    id: formats
+    attributes:
+      label: Formats
+      description: Under which formats is your ontology available? See also OBO Principle [FP-002 "Common Format"](https://obofoundry.org/principles/fp-002-format.html).
+      options:
+        - label: OWL RDF/XML (.owl)
+          required: true
+        - label: OBO (.obo)
+          required: false
+        - label: OBO Graph JSON (.json)
+          required: false
+
+  - type: markdown
+    attributes:
+      value: |
+        # Community
+
+        This section of the ontology request form contains information about how this ontology relates to the greater community
+  - type: textarea
+    id: dependencies
+    attributes:
+      label: Dependencies
+      description: |
+        Please provide a list of ontologies that yours depends on using their OBO Foundry prefixes
+      placeholder: |
```

Comment:
In the final metadata these are lower case, like

https://raw.githubusercontent.com/OBOFoundry/OBOFoundry.github.io/master/ontology/aism.md

Reply:
Done in [706f12a](https://github.com/OBOFoundry/OBOFoundry.github.io/pull/1824/commits/706f12a9b1b82c52f3e4eba17dbf76d1cf1822bd)